Sources/MongoSwift/Client.swift

```diff
@@ -49,6 +60,13 @@ public class MongoClient {
     /// If command and/or server monitoring is enabled, indicates what event types notifications will be posted for.
     internal var monitoringEventTypes: [MongoEventType]?
 
+    /// The read concern set on this client.
+    public var readConcern: ReadConcern {
```

Comment:
why not `public var readConcern: ReadConcern?` ? I wouldn't expect the client to have a default readConcern with a nil level (tested below)

Reply:
Well those languages don't have idiomatic support for optionals, but I also don't feel very strongly about this. To me it seemed like "Why would I ever check client.readConcern.level == nil", to which the answer might be: "Have I set a client-level read concern, or is it the server default?" In which case, checking whether the optional has been set seems to convey that very same info in clearer, more idiomatic fashion.

Reply:
yeah, I guess having to also check for an unset level does sound annoying, will change it!